app/helpers/application_cocoon_helper.rb

```diff
@@ -0,0 +1,53 @@
+module ApplicationCocoonHelper
+  # Shows a link that will allow to dynamically add a new associated object. This is a wrapper of
+  #   {#link_to_add_association}
+  #
+  # @overload link_to_association(name, form, options, association, html_options = nil)
+  #   Shows a link that will allow to dynamically add a new associated object.
+  #   @param [String] name the text to show in the link.
+  #   @param [FormBuilder] form the form builder.
```

Comment:
`ActionView::Helpers::FormBuilder` this type is quite long, so I removed namespaces